packages/prices-api/src/llamma/api.ts

```diff
@@ -17,10 +17,26 @@ export async function getEvents(endpoint: Endpoint, chain: Chain, llamma: string
   }
 }
 
-export async function getTrades(endpoint: Endpoint, chain: Chain, llamma: string, page: number, options?: Options) {
+type GetTradesParams = {
+  endpoint: Endpoint
+  chain: Chain
+  llamma: Address
+  page?: number
+  perPage?: number
+}
+
+export async function getTrades(
+  { endpoint, chain, llamma, page = 1, perPage = 10 }: GetTradesParams,
+  options?: Options,
+) {
   const host = getHost(options)
+  const params = new URLSearchParams({
```

Comment:
calling `toString` in each one is cumbersome and repetitive. In my PR I'm creating a helper method like
```typescript
const paramsToString = (params: Record<string, string | number | boolean>) =>
  new URLSearchParams(Object.entries(params)
    .map(([key, value]) => [key, encodeURIComponent(value)])
  ).toString()

```

Reply:
I'm not sure how `myParams.toString()` is more cumbersome than `paramsToString(myParams)` 🤔